Makefile

```diff
@@ -184,7 +184,8 @@ install : $(INSTALL_DIR) \
 	scripts_install \
 	bin_install \
 	extras_install \
-	data_install
+	data_install \
+	setuid_extract_plugin_snapshot
```

Comment:
This target should call inside

```
extras_install : $(EXTRAS_INSTALL_DIR) install_python_fogbench install_extras_scripts setuid_extract_plugin_snapshot
```

Reply:
@ashish-jabble can you please elaborate upon the shortcomings/errors in the current code and how the proposed code solves/fixes that?

Reply:
As extract_plugin_snapshot is a utility and the part of `extras/`. In _Makefile_ we made as a practice all extras utilities should go under `extras_install`. So that it will clear to all what are the extras we made for foglamp.